src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -285,6 +284,7 @@ ReportActionItemMessageEdit.defaultProps = defaultProps;
 export default compose(
     withLocalize,
     withWindowDimensions,
+    withKeyboardState,
```

Comment:
When I added this HOC, since `ReportActionItemMessageEdit` has it's own forwarded ref, I found out that this HOC isn't properly handling forwarded refs like the `withWindowDimensions` HOC. This caused me to do a refactoring of `withKeyboardState` so that it will better handle the forwarded refs.